Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -137,7 +137,7 @@
       constexpr xy_float_t okay_homing_xy = safe_homing_xy;
     #endif
 
-    destination.set(okay_homing_xy, current_position.z);
```

Comment:
This should be invoking `set(const XYval<T> pxy, const T pz)`, so is there a compiler bug?